ledger/participant-local-store/src/main/scala/com/daml/platform/localstore/InMemoryPartyRecordStore.scala

```diff
@@ -70,7 +69,6 @@ class InMemoryPartyRecordStore(executionContext: ExecutionContext) extends Party
     })
   }
 
-  // TODO um-for-hub: Add a conformance test exercising a race conditions: multiple update on the same non-existing party-record (which exists on the ledger and is indexed by this participant) calls
```

Comment:
That's pretty much impossible to add a party such that its party-record is not created, because it can happen only if participant goes down before receiving successful party-allocated response from the read service